sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/providers/BigQueryStorageWriteApiSchemaTransformProvider.java

```diff
@@ -98,6 +98,15 @@ public String identifier() {
     return String.format("beam:schematransform:org.apache.beam:bigquery_storage_write:v2");
   }
 
+  @Override
+  public String description() {
+    return String.format(
+        "Writes data to BigQuery using the new Storage Write API.\n\nThis expects a single PCollection of Beam Rows "
```

Comment:
"... using the BigQuery Storage Write API" ("new" is relative :) ).

May be also link to https://cloud.google.com/bigquery/docs/write-api